### PR TITLE
Backport claiming stack config

### PR DESCRIPTION
**Pull request by ryaplots**
#### Summary

This PR backports [`257a4ee`](https://github.com/TheThingsIndustries/lorawan-stack/commit/257a4ee99e498efa91a91593a4e3855156408b89) and [`b2b4bbe`](https://github.com/TheThingsIndustries/lorawan-stack/commit/b2b4bbe12acb0cdb4e5553179a17e3977f59172c) from Enterprise

#### Changes

- Add claiming stack config
- Fix duration encoder

#### Testing

cypress

#### Checklist

- [ ] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [x] Documentation: Relevant documentation is added or updated.
- [ ] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: config/stack/ttn-lw-stack-tls.yml [hunk 1]
```diff
@@ -32,6 +32,8 @@ console:
       base-url: 'https://localhost:8885/api/v3'
     gcs:
       base-url: 'https://localhost:8885/api/v3'
+    dcs:
+      base-url: 'http://localhost:1885/api/v3'
```
Comment (adriansmares) [inline config/stack/ttn-lw-stack-tls.yml]: I just saw now that this should be `https` and port `8885`.

**Review by adriansmares — COMMENTED** [review]
Let's also add it in these two places:
https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.22/config/stack/ttn-lw-stack.yml#L94
https://github.com/TheThingsNetwork/lorawan-stack/blob/v3.22/config/stack/ttn-lw-stack-docker.yml#L93

File: config/stack/ttn-lw-stack-docker.yml [hunk 2]
```diff
@@ -92,6 +92,8 @@ tls:
 #       base-url: 'https://thethings.example.com/api/v3'
 #     edtc:
 #       base-url: 'https://thethings.example.com/api/v3'
+#     dcs:
+#       base-url: 'https://localhost:1885/api/v3'
```
Comment (adriansmares) [inline config/stack/ttn-lw-stack-docker.yml]: Use `thethings.example.com`.

File: config/stack/ttn-lw-stack-tls.yml [hunk 3]
```diff
@@ -33,7 +33,7 @@ console:
     gcs:
       base-url: 'https://localhost:8885/api/v3'
     dcs:
-      base-url: 'http://localhost:1885/api/v3'
+      base-url: 'https://localhost:1885/api/v3'
```
Comment (adriansmares) [inline config/stack/ttn-lw-stack-tls.yml]: The port is still `1885`.

File: config/stack/ttn-lw-stack.yml [hunk 4]
```diff
@@ -92,6 +92,8 @@ tls:
 #       base-url: 'https://thethings.example.com/api/v3'
 #     edtc:
 #       base-url: 'https://thethings.example.com/api/v3'
+#     dcs:
+#       base-url: 'https://localhost:1885/api/v3'
```
Comment (adriansmares) [inline config/stack/ttn-lw-stack.yml]: Use `thethings.example.com`.

**Review by adriansmares — APPROVED** [review]

**Review by KrishnaIyer — APPROVED** [review]
👍

**Comment by adriansmares** [comment]
Also needs: 
https://github.com/TheThingsIndustries/lorawan-stack/blob/5b473ccc403820efbc99773bf82a639d7857a58d/sdk/js/src/util/constants.js#L15
https://github.com/TheThingsIndustries/lorawan-stack/blob/5b473ccc403820efbc99773bf82a639d7857a58d/sdk/js/src/util/constants.js#L29

Which were already part of Enterprise so they didn't have an explicit commit in the merge PR.

**Review by kschiffer — APPROVED** [review]

**Pull request merged**